### PR TITLE
Refactor code, fix consultation view commands, add view commands for mastery checks

### LINKED ISSUE #30: Add Mastery Check (MC) session
As a tutor, I can set-up meeting times for Mastery Checks so that I will remember to grade the students for MCs.

### LINKED ISSUE #205: Refactor codebase for clarity in package structure

### LINKED ISSUE #206: Add structure for Consultations and Mastery Check View Commands

### PR DESCRIPTION
**Pull request by wireseo**
Fixes #205 
Fixes #206
Fixes #30

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/view/ViewUngradedMissionCommand.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
-package seedu.address.logic.commands;
+package seedu.address.logic.commands.view;
```
Comment (ngzhenteng) [inline src/main/java/seedu/address/logic/commands/view/ViewUngradedMissionCommand.java:1]: Thanks for the refactor

File: src/main/java/seedu/address/logic/commands/view/ViewUpcomingMasteryChecksCommand.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import seedu.address.model.Model;
+
+/**
+ * Lists only upcoming mastery checks.
+ */
+public class ViewUpcomingMasteryChecksCommand extends ViewCommand {
+    public static final String MESSAGE_SUCCESS = "Listed upcoming mastery check sessions";
+
+    @Override
+    public CommandResult execute(Model model) {
```
Comment (ngzhenteng) [inline src/main/java/seedu/address/logic/commands/view/ViewUpcomingMasteryChecksCommand.java:16]: Method looks great

File: src/main/java/seedu/address/logic/parser/ViewCommandParser.java [hunk 3]
```diff
@@ -97,6 +109,23 @@ public ViewCommand parse(String args) throws ParseException {
         case VIEW_DEADLINE_LIST:
             return new ViewTaskListCommand(VIEW_DEADLINE_LIST);
 
+        case VIEW_CONSULTATION_LIST:
```
Comment (ngzhenteng) [inline src/main/java/seedu/address/logic/parser/ViewCommandParser.java:112]: Looks great, maybe we need to extract some cases out later, to reduce the code to beneath 30loc.

**Review by ngzhenteng — COMMENTED** [review]
LGTM, thank you for the extensive work!

File: src/main/java/seedu/address/logic/parser/ViewCommandParser.java (hunk 3 above)
Comment (wireseo) [reply src/main/java/seedu/address/logic/parser/ViewCommandParser.java:112]: Good idea!

**Pull request merged**